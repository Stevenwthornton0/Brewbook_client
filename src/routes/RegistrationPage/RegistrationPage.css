.registrationPage {
    display: flex;
    flex-direction: column;
    margin-right: auto;
    margin-left: auto;
    width: fit-content;
}

.registrationPage > h2 {
    font-size: 26px;
    font-weight: 400;
    margin-top: 50px;
    margin-bottom: 16px;
}

@media screen and (max-width: 500px) {
    .registrationPage > h2 {
        margin-bottom: 0;
        margin-top: 40px;
        align-self: center;
    }
}