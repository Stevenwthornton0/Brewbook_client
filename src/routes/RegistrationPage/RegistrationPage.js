import React, { Component } from 'react';
import RegistrationForm from '../../components/RegistrationForm/RegistrationForm';
import './RegistrationPage.css'

class RegistrationPage extends Component {
    static defaultProps = {
        histor: {
            push: () => {}
        }
    }

    handleRegistrationSuccess = user => {
        const { history } = this.props;
        history.push('/')
    }

    render() {
        return (
            <section className='registrationPage'>
                <h2>Register</h2>
                <RegistrationForm 
                    onRegistrationSuccess={this.handleRegistrationSuccess}
                />
            </section>
        )
    }
}

export default RegistrationPage;