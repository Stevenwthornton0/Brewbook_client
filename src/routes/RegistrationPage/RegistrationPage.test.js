import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme from 'enzyme';
import RegistrationPage from './RegistrationPage';

Enzyme.configure({ adapter: new Adapter() })

describe(`RegistrationPage smoke test`, () => {
    it(`renders without crashing`, () => {
        const wrapper = shallow(<RegistrationPage />)
        expect(toJson(wrapper)).toMatchSnapshot()
    })
})