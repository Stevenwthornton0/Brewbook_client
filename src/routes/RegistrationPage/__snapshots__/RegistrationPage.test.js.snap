// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegistrationPage smoke test renders without crashing 1`] = `
<section
  className="registrationPage"
>
  <h2>
    Register
  </h2>
  <RegistrationForm
    onRegistrationSuccess={[Function]}
  />
</section>
`;
